CHANGELOG.md

```diff
@@ -8,13 +8,43 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- Added process sets for TensorFlow: Concurrently running collective operations on subsets of Horovod processes. ([#2839](https://github.com/horovod/horovod/pull/2839))
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+## [v0.23.0] - 2021-10-06
+
+### Added
```

Comment:
Where is this change?
```
### Removed
Spark/Keras: remove bare Keras support. (#3191)
```

Reply:
It was already added in a previous commit, line 53.